mlir/utils/performance/perfRunner.py

```diff
@@ -1140,63 +1165,75 @@ def benchmarkFusionKernels(test_dir, paths: Paths, arch, numCU, tuningDb: MaybeT
                 splitPerf[6] = '1'
                 tuningDb[arch,config] = ','.join(splitPerf)
 
-    # Profile each test case
-    for test in allTests:
-        filename = test['filename']
-        testVector = test['testVector']
-        futName = test['futName']
-
-        print("Profiling:", filename)
-        # Sanity check
-        if not testVector:
-            print("\tCannot find a test vector")
-            continue
-        if not futName:
-            print("\tCannot find rocmlir-gen with -fut")
-            continue
+    # Prepare test cases
```

Comment:
Merged the two loops into one, with the split-k hack moved up first.  Go through each .mlir file, extract the useful RUN: command if present, and do makeBasicFusionPipeline with it to make the initial mlir code.  We have a temp file to hold that code, because I couldn't make a good way to save it as a string and then send it to another pipeline later without having a file.